Installing-and-Configuring/OS/index.md

```diff
@@ -58,17 +58,17 @@ Your other partitions may be encrypted or not based on your preferences and this
+Heads can protect a LUKS Disk Unlock Key using the TPM.  This will ensure that upon booting with Heads only a TPM disk unlock passphrase and the proper PCR state can access the data on the root partition.  In this model the TPM enforces rate limits to prevent brute force attacks.  If the disk is moved to a different system the [LUKS disk unlock passphrase/recovery key](/Keys/#tpm-disk-encryption-key) in slot 0 may be used to access the encrypted data.  This model is similar to the [Security Dongle](#security-dongle-with-luks-root) model.
 
-If this feature is enabled, heads will ask for a Disk Unlock Passphrase when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The Disk Unlock passphrase will be used to generate a Disk Unlock Key to be placed in the second LUKS keyslot for the root partition.  The disk unlock key is sealed in the TPM along with measurements taken at that time.
+If the Heads TPM drive decryption feature is enabled, Heads will [ask for](/Keys/#tpm-disk-encryption-key) a Disk Unlock Passphrase (which might be a numerical PIN) when saving the default boot partition configuration.  This is not the same as the passphrase/key used by LUKS.  The TPM Disk Unlock passphrase will be used to generate a LUKS Disk Unlock Key to be placed in keyslot 1 for the root partition.  The LUKS disk unlock key is sealed in the TPM along with measurements taken at that time.
 
-On boot, heads will ask for the disk unlock passphrase that will release the Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the Key was created.  
+On boot, Heads will ask for the TPM disk unlock passphrase that will release the LUKS Disk Unlock Key.  This will only succeed if the current measurements match the ones taken when the keys were created.  
 
-This option adds an extra layer of security.  The TPM will only release the Disk Unlock key if the measurements match and generally an attacker would not have the Disk Unlock passphrase.   If an attacker clones the drive and compromises the disk unlock passphrase he will still not have access to the decrypted data.  This is due to this passphrase being valid only on this laptop with this firmware and these measured files.  
+This option adds an extra layer of security.  The TPM will only release the LUKS Disk Unlock key if the measurements match.  Generally, an attacker would not have the LUKS Disk Unlock key in keyslot 1 because it was generated by Heads and would not have access to the LUKS disk unlock passphrase in keyslot 0 because it was only used at install.  Even if the attacker clones the drive he will still not have access to the decrypted data.  
```

Comment:
This option adds an extra layer of security. The TPM will only release the LUKS Disk Unlock key if the measurements match (again, LUKS header, firmware measurements and Disk Unlock Key passphrase releasing TPM VN stored Disk Unlocked key if everything is congruent). 

Generally, an attacker would not have the LUKS Disk Unlock key in key slot 1, because it was generated by Heads and would not have access to the LUKS Disk Recovery Key passphrase stored under keyslot 0 because it was only used at install, and provided when defining new boot default to nenew/change the Disk Unlock Key passphrase. Even if the attacker clones the drive he will still not have access to the decrypted data, unless he eavesdropped the Disk Recovery Key passphrase.